cdc/sinkv2/eventsink/txn/mysql/mysql.go

```diff
@@ -342,9 +347,6 @@ func (s *mysqlBackend) execDMLWithMaxRetries(ctx context.Context, dmls *prepared
 					start, s.changefeed, "COMMIT", dmls.rowCount, dmls.startTs)
 			}
 
-			for _, callback := range dmls.callbacks {
```

Comment:
Removing callback to outside will increase the throughput ?

Reply:
It's for the new metric `TxnSinkDMLBatchCallback`.